tests/UI/pages/FO/FObasePage.js

```diff
@@ -208,7 +242,19 @@ module.exports = class FOBasePage extends CommonPage {
    * @returns {Promise<void>}
    */
   async goToCartPage(page) {
-    await this.clickAndWaitForNavigation(page, this.cartLink);
+    await this.goToHeaderLink(page, 'Cart');
+  }
+
+  /**
+   * Search product
+   * @param page
+   * @param productName
+   * @returns {Promise<void>}
+   */
+  async searchProduct(page, productName) {
+    await this.setValue(page, this.searchInput, productName);
+    await page.keyboard.press('Enter');
+    await page.waitForNavigation();
```

Comment:
Please set the navigation method here 'networkidle' or 'load' or 'domcontentloaded'

Reply:
Done thx :+1: